app/test/unit/endpoint-capabilities-test.ts

```diff
@@ -9,45 +11,46 @@ import { forceUnwrap } from '../../src/lib/fatal-error'
 describe('endpoint-capabilities', () => {
   describe('endpointSatisfies', () => {
     it('recognizes github.com', () => {
-      expect(testDotCom(true)).toBeTrue()
-      expect(testDotCom(false)).toBeFalse()
+      assert.equal(testDotCom(true), true)
+      assert.equal(testDotCom(false), false)
     })
 
     it('recognizes GHES', () => {
-      expect(testGHES(false)).toBeFalse()
-      expect(testGHES(true)).toBeTrue()
+      assert.equal(testGHES(false), false)
```

Comment:
Nit: This file has a lot of opportunity for `assert(condition)` as opposed to the more verbose `assert.equal(condition, boolean)`